bindata/network/network-metrics/001-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metrics-daemon-sa
+  namespace: openshift-network-metrics
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: metrics-daemon-role
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["network-attachment-definitions"]
+    verbs: ["get", "watch", "list"]
+  - apiGroups: ["authentication.k8s.io"]
+    resources: ["tokenreviews"]
+    verbs: ["create"]
+  - apiGroups: ["authorization.k8s.io"]
+    resources: ["subjectaccessreviews"]
+    verbs: ["create"]
```

Comment:
why does it need tokenreviews and subjectaccessreviews create permission?

Reply:
It's because of the kube-rbac-proxy. We are using it for exposing the metrics endpoint securely, but it also requires those permissions as it performs TokenReviews and SubjectAccessReviews. I followed the same approach as https://github.com/openshift/cluster-network-operator/pull/563 and other operators, putting the rbac-proxy in the pod.